src/sscce.js

```diff
@@ -47,6 +47,12 @@ module.exports = async function(createSequelizeInstance, log) {
     const User = sequelize.define('User', {
         name: DataTypes.TEXT,
         pass: DataTypes.TEXT
+    }, {
+        hooks: {
+            afterCreate: (user, options) => {
+                asynchronusReading()

```

Comment:
Let me know if the following change removes the warning

```suggestion
                return asynchronusReading()
```

Reply:
Thanks @papb,
returning the `asynchronusReading()` solved my issue.